foc.h

```diff
@@ -22,6 +22,13 @@ class FOC {
     void set_iq_limit(float limit) { iq_limiter_.set_limit(limit*dt_); }
     float get_id_limit() const { return id_limiter_.get_limit()/dt_; }
     float get_iq_limit() const { return iq_limiter_.get_limit()/dt_; }
+    bool is_voltage_saturated() const { 
+      return pi_id_.is_voltage_saturated() || pi_iq_.is_voltage_saturated();
```

Comment:
Do we need to be checking the combination of voltage output of both of these two? I thought iq and id might have additive voltages at the phases, but I'm not an FOC expert